.github/workflows/nightlies.yml

```diff
@@ -18,6 +18,8 @@ on:
 jobs:
   # Build and package a new release for all supported platforms
   build-all-releases:
+    permissions:
+      contents: none
```

Comment:
Why `none`? This action clones this repo, along with druntime and phobos. If I understand correctly, `read: none` means that this action won't be able to access the contents of this and other repositories (but will likely instead hit the GH API rate limit.). Also, if use intended to disable all permissions, `permissions: {}` would be a more understandable way of doing it.

Reply:
I dont think rate limit is a problem on public git clones inside Github Actions since it doesn't use the normal GH API. I might be wrong.

Reply:
ok. I think I got that wrong.

Reply:
Change this to content read, as its enough and don't have any security concerns 🙂.